examples/test/videosphere/index.html

```diff
@@ -9,15 +9,32 @@
   <body>
     <a-scene stats>
       <a-assets>
-        <video id="lego" preload="true"
-               src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"
+        <!-- Test anonymous video with source element. -->
+        <!-- Normally, one would give the video an id, e.g. id="lego" -->
+        <video preload="true"
                width="360" height="360" autoplay loop="true"
-               crossOrigin="anonymous"></video>
+               crossOrigin="anonymous">
+          <source src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"></source>
```

Comment:
I'm getting this error in macOS desktop Safari:

![image](https://user-images.githubusercontent.com/203725/31714869-6ed6865e-b3b6-11e7-8ddd-1a4370632da8.png)

(all the other video examples seem to work in Safari when I have `Develop > Cross-Origin Restrictions` enabled.)

Reply:
same on iOS Safari too:

![image](https://user-images.githubusercontent.com/203725/31714960-d6714d9e-b3b6-11e7-9ab3-907ff248d34a.png)

Reply:
does Safari support webm format?  I assumed that was why this example fails there

Reply:
(That failure should be present with or without this PR.)